src/main/java/com/google/sps/data/GraphDataObject.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GraphDataObject {
+  List<Object> hourList = new ArrayList<>();
+  List<Object> difficultyList = new ArrayList<>();
+
+  public GraphDataObject(String hourTitle, String diffTitle) {
+    List<String> titleContainer = new ArrayList();
+    titleContainer.add(hourTitle);
+    hourList.add(titleContainer);
```

Comment:
If you're just needing to turn "hourTitle" into an array to add it to the list then I'm not sure you need to create the "titleContainer" variable to do that, seems you could just do this in place here.

Reply:
Same comment below for addHour and addDifficulty